bin-src/lib/tasks.ts

```diff
@@ -15,6 +15,8 @@ export const createTask = ({ title, steps, ...config }): Listr.ListrTask<Context
       // eslint-disable-next-line no-await-in-loop
       await step(ctx, task);
     }
+
+    ctx.options.onTaskComplete?.(ctx);
```

Comment:
```suggestion
    ctx.options.onTaskComplete?.({ ...ctx });
```
This should ensure `ctx.title` still refers to the title of the current (finished) task rather than the next one.